Dockerfile

```diff
@@ -20,6 +20,14 @@ RUN if [ $(arch) = x86_64 ]; then \
     kernel-rt-modules-extra${RT_KERNEL_VERSION:+-}${RT_KERNEL_VERSION}; \
     fi
 
+# 64k-pages kernel packages for aarch64
+RUN if [ $(arch) = aarch64 ]; then \
+    dnf -y install \
+    kernel-64k-devel${KERNEL_VERSION:+-}${KERNEL_VERSION} \
+    kernel-64k-modules${KERNEL_VERSION:+-}${KERNEL_VERSION} \
+    kernel-64k-modules-extra${KERNEL_VERSION:+-}${KERNEL_VERSION}; \
```

Comment:
Don't we need `kernel-64k-headers` as well?

Reply:
The `kernel-64k-headers` doesn't exist. Only the packages with binaries (kernel and modules) need to be compiled with 64k pages. The headers are provided by the `kernel-headers` packages.

Reply:
That makes sense. Can you please add a comment in the dockerfile saying something like:
"header files aren't compiled, therefore for 64k pages kernel we will use the `kernel-headers-<version>` package"
?

Reply:
Done.